tests/Paket.Tests/PackageProcessSpecs.fs

```diff
@@ -11,7 +11,7 @@ let ``Loading assembly metadata works``() =
     let workingDir = Path.GetFullPath(".")
     
     let fileName =
-        Path.Combine(workingDir, "..", "..", "Paket.Tests.fsproj")
+        Path.Combine(workingDir, "Paket.Tests.fsproj")
```

Comment:
Hmm interesting, I don't get that on Mac or Windows, maybe there's something off with my setup.